Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -515,7 +516,31 @@ namespace Multiplayer
          || GetAgentType() == MultiplayerAgentType::DedicatedServer)
         {
             // We use a temporary userId over the clients address so we can maintain client lookups even in the event of wifi handoff
-            NetworkEntityHandle controlledEntity = SpawnDefaultPlayerPrefab(packet.GetTemporaryUserId());
+            IMultiplayerSpawner* spawner = AZ::Interface<IMultiplayerSpawner>::Get();
+            NetworkEntityHandle controlledEntity;
+
+            // Check rejoin data first
+            const auto node = m_playerRejoinData.find(packet.GetTemporaryUserId());
```

Comment:
Hrm.. this was for multihost player migration.. the code here doesn't need to change since this is all negotiated at a higher level in a different GEM; but now that you're writing a formalized interface that would allow game specific rejoin behaviour I should probably rename some of these members to make things less confusing.